src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/workflow_enum.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Fixtures\Workflow\Places;
+use Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\FrameworkExtensionTestCase;
+
+$container->loadFromExtension('framework', [
+    'workflows' => [
+        'enum' => [
+            'supports' => [
+                FrameworkExtensionTestCase::class,
+            ],
+            'places' => Places::cases(),
+            'transitions' => [
+                'one' => [
+                    'from' => Places::A->value,
+                    'to' => Places::B->value,
```

Comment:
```suggestion
                    'from' => Places::A,
                    'to' => Places::B,
```

In a following PR ?

Reply:
May be 🤞🏼

Reply:
implemented

Reply:
I updated the test while at it in #61755